ctapipe/io/containers.py

```diff
@@ -540,6 +546,8 @@ class DataContainer(Container):
         EventCalibrationContainer(),
         "Container for calibration coefficients for the current event"
     )
+    weather = Field(Map(WeatherContainer),
```

Comment:
why is this a map?

Reply:
I think just a `Field(WeatherConainer(), "weather information")` would be fine

Reply:
Oh, I see, the Map() is thought to be used for repeatedly (for several telescopes) used sub-containers, which isn't necessary here. So no reason to use "Map", I assume.